frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -548,15 +546,10 @@ const ManageHostsPage = ({
     }
 
     const { MANAGE_HOSTS } = PATHS;
-    const isAllHosts = slug === ALL_HOSTS_LABEL;
+    const isAllHosts = !status;
     const newFilters = [...selectedFilters];
 
     if (!isAllHosts) {
```

Comment:
Could use an extra set of eyes if this `handleLabelChange` function properly removes the use of status slugs